Comp2024/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -30,11 +32,12 @@
  */
 public class CommandSwerveDrivetrain extends SwerveDrivetrain implements Subsystem
 {
-  private final boolean                          m_useLimelight = false; // set to false when no limelight to prevent sim errors
+  // private final boolean                          m_useLimelight = true; // set to false when no limelight to prevent sim errors
 
   private static final double                    kSimLoopPeriod = 0.005; // 5 ms
   private Notifier                               m_simNotifier  = null;
   private double                                 m_lastSimTime;
+  private final boolean                          m_useLimelight = true; // set to false when no limelight to prevent sim errors
```

Comment:
When checking in the code, we need to leave this as "false" to prevent everyone else from having simulation problems. When we get working robots, we can leave it as "true". You'll need to set it to true when you are testing.